pkg/cluster/cluster.go

```diff
@@ -21,9 +27,27 @@ func New(logger micrologger.Logger, runner runner.Runner, remoteCluster bool) *C
 	}
 }
 
-// Create is a Task that creates a remote cluster.
+// Create is a Task that creates a remote cluster or, if we
+// are using a local one, puts in place the required files for
+// later access to it
 func (c *Cluster) Create() error {
-	return c.clusterAction("shipyard -action=start")
+	if c.remoteCluster {
+		return c.clusterAction("shipyard -action=start")
+	}
+	usr, err := user.Current()
+	if err != nil {
+		return err
+	}
+
+	err = c.copyMinikubeAssets(usr.HomeDir)
+	if err != nil {
+		return err
+	}
+	err = c.setupMinikubeConfig(usr.HomeDir)
+	if err != nil {
+		return err
```

Comment:
I would like to see error masking everywhere.

Reply:
Agree, i'll start by masking every error once in the closest place to where it originated, we can extend from there if needed.

Reply:
IMO only masking in the originating place is just not good enough. You will refactor code and forget to mask where the origin changes. You will also not see where the error is going through and when there are multiple options you will have a harder time to find out what is actually going on. As I said already. Feel free to suggest alternatives to the state of the art but as long as there is no alternative that meets all the requirements of debuggability I would like to stick with the current conventions. 

It was just on Saturday that I was on call and had to debug gsctl, which does not mask errors. It took me 30 minutes during an incident situation to find out what the fucking problem was and where the error came from. Considering that I am always asking to mask errors everywhere you can imagine that I was pretty pissed on the weekend when I had to dig into the depths of some project I am not used to.

Reply:
> You will also not see where the error is going through and when there are multiple options you will have a harder time to find out what is actually going on

I agree with the convention but could you pls explain this? I feel I'm missing something, you get an error and it has attached the stack trace of where it was originated. All the calls that provoked that error are recorded there with file names and line numbers, what is the advantage of masking the error again as it bubbles up and the call stack unrolls?

> It was just on Saturday that I was on call and had to debug gsctl, which does not mask errors

Indeed, that's a bad situation, not masking at all should not happen.